FWCore/Framework/interface/TriggerResultsBasedEventSelector.h

```diff
@@ -29,6 +31,13 @@ namespace edm {
     public:
       NamedEventSelector(std::string const& n, EventSelector const& s) :
```

Comment:
@Dr15Jones No, it should not be needed. However, the optimal fix to get rid of it invlves other changes, and will be done in a separate PR, to avoid having to rebuild > 800 packages when testing.